onnxruntime/core/session/onnxruntime_c_api.cc

```diff
@@ -49,7 +49,7 @@ ProviderInfo_CUDA* TryGetProviderInfo_CUDA();
 
 #ifdef ENABLE_TRAINING_APIS
 #include "orttraining/training_api/include/onnxruntime_training_c_api.h"
-#include "orttraining/training_api/include/ort_training_apis.h"
+#include "orttraining/training_api/ort_training_apis.h"
```

Comment:
Normally, include is placed level above after orttraining. This way, the include would continue to look like `#include training_api/ort_training_api.h`

Or you could simply make it like `#include training/ort_training_api.h`

This would be according to established standards like `gsl/gsl`